coral/docs/development-with-local-klaw.md

```diff
@@ -22,6 +22,7 @@ Please check out the [proxy README](../proxy/README.md) for more detailed inform
 
 - [node](https://nodejs.org/en/) needs to be installed. See [nvmrc](../.nvmrc) or the `engines` definition in [package.json](../package.json) for version.
 - Coral uses [pnpm](https://pnpm.io/) (version 7) as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) pnpm.
+- [Maven](https://maven.apache.org/) is the build tool used for Klaw and needs to be installed. You can [download the binary](https://maven.apache.org/download.cgi) or use a package manager (like [Homebrew](https://formulae.brew.sh/formula/maven) on MacOs).
 
```

Comment:
In fact there is no need for `maven` installed. 
A user could use maven wrapper which is a part of this project `./mvnw` in case of Linux/Mac or `mvnw.bat` in case of Windows like
```
./mvnw clean verify
```
or
```
mvnw.bat clean verify
```
it will check if there is an existing maven on the system. In case of success it will use it.
In case of failure it will download it from the url mentioned at `.mvn/wrapper/maven-wrapper.properties`